### ISSUE TITLE
[enhancement] Chapter selection

### ISSUE THREAD

**Issue by big-cat-loaf** [TagoDR/MangaOnlineViewer#398]
Most of the Manga website have a Chapter selection, when you want to jump more than one chapter. This is an example form www.mgeko.com: 
![Screenshot 2023-12-27 153325](https://github.com/TagoDR/MangaOnlineViewer/assets/100969549/a33a8c01-9fd0-44ff-9ae8-382839ebe734)
Can you intercrate in your script?

**Comment by big-cat-loaf** [comment]
Have you decided after a year if you want to implement it?

**Comment by TagoDR** [comment]
I dont think this would be used much, is almost always better to just go the chapter list.
It would require analysis of every site to make something universal, if even possible.
No one stepped up to help, so I'm closing this
